### PR TITLE
Iros 2021 pubs

### PR DESCRIPTION
**Pull request by calebescobedo**
Add commends on what information should be added or changed regarding the two new publications.

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
+ - title: "Contact Anticipation for Physical Human-Robot Interaction with Robotic Manipulators using Onboard Proximity Sensors"
+   authors: Caleb Escobedo*, Matthew Strong, Mary West, Ander Aramburu, Alessandro Roncone
```
Comment (peasant98) [inline _data/publications.yml:2]: I don't think you need the * by your name here

File: _data/publications.yml [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+   year: 2021
+   venue: IEEE/RSJ International Conference on Intelligent Robots and Systems [IROS]
+   where: Prauge, Czech Republic, Sept 27-Oct 1
```
Comment (peasant98) [inline _data/publications.yml:5]: Prauge --> Prague

Comment (peasant98) [inline _data/publications.yml:5]: Sept and Oct should be written as the full month names

Comment (peasant98) [inline _data/publications.yml:5]: Super nit: I'd put "The Czech Republic"

File: _posts/people/2019-10-10-caleb.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ pubyears:
  - all
 
 cv:
+ - name: publications
+   elems:
+    - title: "Contact Anticipation for Physical Human-Robot Interaction with Robotic Manipulators using Onboard Proximity Sensors"
+      authors: Caleb Escobedo*, Matthew Strong, Mary West, Ander Aramburu, Alessandro Roncone
```
Comment (peasant98) [inline _posts/people/2019-10-10-caleb.md:16]: same comments apply from above with the formatting things

File: _posts/news/2021-07-01-IROS-2021.md [hunk 4]
```diff
@@ -6,4 +6,15 @@ author: Alessandro Roncone
 ---
 
 Congratulations to [Caleb]({% post_url people/2019-10-10-caleb %}), Kandai, Matt, Mary, Ander and Krishna for their paper acceptance at IROS 2021!
+
+
+#### _Contact Anticipation for Physical Human-Robot Interaction with Robotic Manipulators using Onboard Proximity Sensors_ by [Caleb Escobedo]({% post_url people/2019-10-10-caleb %}), <a href="https://www.linkedin.com/in/matthewhstrong/">Matthew Strong</a>, Mary West, Ander Aramburu, and Alessandro Roncone
+
+This publication introduces dynamic contact thresholding, a method for contact anticipation during physical human-robot interaction in close proximity dynamic environments. Additionally, a framework is introduced that outlines object detection, prior to contact behavior, contact detection, and post contact behavior for a robot manipulator.
```
Comment (peasant98) [inline _posts/news/2021-07-01-IROS-2021.md:13]: I feel that you should introduce the framework, then say how dynamic contact thresholding is the core element of it.

File: _posts/news/2021-07-01-IROS-2021.md [hunk 5]
```diff
@@ -6,4 +6,15 @@ author: Alessandro Roncone
...
+
+
+#### _Self-Contained Kinematic Calibration of a Novel Whole-Body Artificial Skin for Human-Robot Collaboration_ by Kandai Watanabe, <a href="https://www.linkedin.com/in/matthewhstrong/">Matthew Strong</a>, Mary West, [Caleb Escobedo]({% post_url people/2019-10-10-caleb %}), Ander Aramburu, Krishna Chaitanya, and Alessandro Roncone
+
+The paper presents a kinematic calibration algorithm that can automatically locate sensors placed arbitrarily on the surface of a robot manipulator given they are equipped with an Inertial Measurement Unit (IMU). To evaluate this calibration method and demonstrate potential applications, the HIRO group developed a stand-alone artificial skin unit equipped with an IMU and proximity sensor for object detection.
```
Comment (peasant98) [inline _posts/news/2021-07-01-IROS-2021.md:18]: place a comma after "robot manipulator"

File: _posts/people/2019-10-10-caleb.md [hunk 6]
```diff
@@ -46,13 +58,7 @@ cv:
       end: Dec 2018
       location: San Antonio, TX, United States
       description: "I was a teaching assistant for two classes, Algorithms and Data Abstraction. While in this position I held office hours, evaluated student performance, and gave C++ demonstrations."
- - name: publications
```
Comment (peasant98) [inline _posts/people/2019-10-10-caleb.md:49]: I think you should still include the BS thesis in publications

**Review by peasant98 — CHANGES_REQUESTED** [review]

File: _posts/news/2021-07-01-IROS-2021.md (hunk 4 above)
Comment (calebescobedo) [reply _posts/news/2021-07-01-IROS-2021.md:13]: I think that would make it seem like the framework is more important than the dynamic contact thresholding. I'm thinking of a way to change the wording.

Comment (peasant98) [reply _posts/news/2021-07-01-IROS-2021.md:13]: Ah, sounds good

**Comment by alecive** [comment]
Merging in the interest of time. Feel free to push to this branch and open a new PR to fix the issues above!

**Pull request merged**